templates/problem/editorial.html

```diff
@@ -21,7 +21,7 @@
         {% if not has_solved_problem %}
             <div class="alert alert-danger">{{ _('Remember to use this editorial <b>only</b> when stuck, and <b>not to copy-paste code from it</b>. Please be respectful to the problem author and editorialist. <br><br> <b>Submitting an official solution before solving the problem yourself is a bannable offence.</b>') }}</div>
         {% endif %}
-        {% if solution.authors.exists() %}
+        {% if solution.authors.all() %}
```

Comment:
What @quantum5 was referring to is something like https://github.com/DMOJ/site/blob/88c1aa319250444230589ade04aef2e8bd68a6b9/templates/problem/problem.html#L175

As-is, this still performs 2 queries.